Project-II-Calculator/calculator/src/components/ButtonComponents/ActionContainer.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ActionButton from './ActionButton';
+
+const buttons = [
+    {action: '÷'},
+    {action: 'x'},
+    {action: '-'},
+    {action: '+'},
+    {action: '='},
+]
+
+function ActionContainer() {
+    const ActionColumn = buttons.map(button => <ActionButton text={button.action} style="button actions"/>);
```

Comment:
this is great! I like how to break it up here for cleaner code later in the return